phlex_ui.gemspec

```diff
@@ -12,10 +12,11 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = ">= 3.2"
 
-  s.add_dependency "phlex", "~> 1.11"
+  s.add_dependency "phlex", "~> 1.10"
   s.add_dependency "rouge", "~> 4.2.0"
   s.add_dependency "zeitwerk", "~> 2.6"
   s.add_dependency "activesupport", ">= 6.0"
+  s.add_dependency "tailwind_merge", "0.12.0"
```

Comment:
@iseth , I don't think we should be so restrictive and lock onto a specific version.

Take [avo](https://github.com/avo-hq/avo/blob/main/avo.gemspec) for example; they are taking an even more optimistic approach and using >= instead of ~> for their dependencies. I believe we should do the same here

Reply:
Good point. Thanks